swagger2locustio/__main__.py

```diff
@@ -14,14 +14,27 @@ def main():
     """Launching function"""
 
     parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument("--swagger-file", help="path to swagger file", required=True, type=Path)
+    parser.add_argument("-f", "--swagger-file", help="path to swagger file", required=True, type=Path)
     parser.add_argument(
-        "--results-path", help="path to store locustfile.py", required=False, default=Path("generated"), type=Path,
+        "-r",
+        "--results-path",
+        help="path to store locustfile.py",
+        required=False,
+        default=Path("generated"),
+        type=Path,
     )
     parser.add_argument(
-        "--loglevel", help="logging level", required=False, default="info", choices=settings.LOGGING_LEVELS, type=str,
+        "-v",
+        "--verbose",
+        nargs="*",
+        help="logging level",
+        required=False,
+        default="info",
+        choices=settings.LOGGING_LEVELS,
```

Comment:
I think that users shouldn't give level arg after --verbose. It should itself means that logging level set to debug.

Reply:
If no argument is passed to --verbose it sets to DEBUG